text/2021-02-24-coprocessor-plugin.md

```diff
@@ -0,0 +1,145 @@
+# Coprocessor Plugin
+
+## Summary
+
+Add a general and plugable coprocessor framework for RawKV mode.
+
+## Motivation
+
+TiKV is the storage component in the TiDB ecosystem, however, the distribution computation principle suggests that computation should be as close to the data source as possible. Therefore, TiKV has embedded a subset of the TiDB executor framework to push down some computation tasks when applicable.
+
+But TiKV's capability should be far beyond that, as many distributed components can be built on top of TiKV, such as cache, full text search engine, gragh database and NoSQL database. And same as TiDB, these product will also like to push down specific computation to TiKV, which requires the coprocessor to be customizable, aka plugable.
+
+## Detailed design
+
+### Dynamic vs statically
+
+Generally, there are two strategies to build a plugin framework: dynamically and statically, which means to load the plugin on startup or to embed in the binary on compilation.
+
+![Plugin Arch](../media/plugin-arch.png)
```

Comment:
`plugin-arch.png` seems to be an empty file.

Reply:
I can see it on https://github.com/tikv/rfcs/blob/c180a9a4e1b5b5515b31a7fb7d77844ef707b2d9/text/2021-02-24-coprocessor-plugin.md